Scripts/CommonServerPython/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-
+  - Added arg_to_boolean command
+  - Added csv_or_array_to_array command
```

Comment:
Remove this line

Reply:
@anara123 To be clear, are you asking me just to remove a single blank line after `## [Unreleased]` or do you also want those lines added?

Reply:
I mean remove `Added csv_or_array_to_array command` because you only implemented arg_to_boolean in this PR

Reply:
Sorry -- for some reason my local repo didn't show either of those two lines when I looked at it -- must've had the wrong commit selected or something.  Anyway, all done.